task/clamav-scan/0.1/README.md

```diff
@@ -0,0 +1,21 @@
+# clamav-scan task
+
+## Description:
+Clam-Antivirus will extract compiled code to compare it against the latest virus database.
```

Comment:
We could also mention here that the ClamAV utility is used and link to its documentation page (https://docs.clamav.net/).

Reply:
I linked to the documentation in my the most recent patch. I did try to keep the descriptions as succinct as I could, within a line or two. We could definitely create more beefier pages with more details of the underlying tooling behind the tasks, especially the more intense ones like clam-av, but I didn't want to attempt to recreate the docs already available. My thinking was to just provide some context at a glance.

Reply:
Thanks!